meeting-minutes/team-minutes/feb10-team-minutes.md

```diff
@@ -0,0 +1,27 @@
+## Before Team Meeting Things To Do:
+- 
+
+## Agenda Items:
+- [aleks] when clicking logout, the user is not directed anywhere. The user stays on the mentor tutorial screen even though they are already logged out.
+- [aleks] on the login screen, the forgot password does not work, it just brings the user to a blank page
+- [aleks] on the login screen, create an account brings the user to Register organization, not to register mentor/mentor form

```

Comment:
That's correct :) Mentors should only be able to view the Mentor form if an admin invites them through an email.